### PR TITLE
Return 404 if repo config doesnt exist

### PR DESCRIPTION
**Pull request by g4s8**
#74 - added check if config exist and return 404 HTTP code if does not

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#117](https://github.com/artipie/artipie/issues/117) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/f62b6a01-bbb2-4235-ab51-267ef60e1ac5); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artipie/Pie.java [hunk 1]
```diff
@@ -83,15 +87,38 @@ public Response response(final String line, final Iterable<Map.Entry<String, Str
...
+                            slice = new AsyncSlice(
+                                storage.value(key).thenApply(
+                                    content -> new SliceFromConfig(
+                                        new RepoConfig(this.vertx, content),
+                                        this.vertx.fileSystem()
+                                    )
+                                )
+                            );
+                        } else {
+                            slice = new SliceSimple(
```
Comment (Vatavuk) [inline src/main/java/com/artipie/Pie.java:108]: @g4s8 can we test this scenario?

**Comment by g4s8** [comment]
@Vatavuk right, added todo for test

**Comment by Vatavuk** [comment]
@g4s8 please remove this issue from the scope

**Comment by g4s8** [comment]
@0crat out

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@Vatavuk please make sure you will find at least 3 issues pruning next CR, thanks

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**